internal/fs/inode/file.go

```diff
@@ -189,8 +189,9 @@ func (f *FileInode) openReader(ctx context.Context) (io.ReadCloser, error) {
 	rc, err := f.bucket.NewReader(
 		ctx,
 		&gcs.ReadObjectRequest{
-			Name:       f.src.Name,
-			Generation: f.src.Generation,
+			Name:           f.src.Name,
+			Generation:     f.src.Generation,
+			ReadCompressed: f.src.ContentEncoding == "gzip",
```

Comment:
can we create constant for "gzip" or wrap this check in a method of object/minObject like IsGzip ?

Reply:
Good idea. Added a function `HasContentEncodingGzip()` now.